### DISCUSSION TITLE
Limit RCF audio to play one time when many messages post simultaneously

### DISCUSSION THREAD

**Discussion by JJWatMyself** [f1multiviewer/issue-tracker#141] [Feature requests] [answered]
RCM audio is awesome! When the noise plays you instantly know to look.  When there are a string of 5 RCM messages, there isn't a need to play the tone five times.  Once is enough.  So the suggestion is to play the noise no more than one time for messages in an amount of time, e.g. 2 seconds.

**Comment by JJWatMyself** [comment]
Method that would use time stamps.

init last_time_tone_played = some old time value
When new RCM
print RCM message to the RCM window
If RCM time > 2 seconds + last_time_tone_played
Then
  set last_time_tone_played = RCM time
  play tone
Else
  do not play tone

**Comment by f1multiviewer** [comment, marked as answer]
This has been implemented!
